keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -43,7 +43,11 @@ class DeliveryConfigController(
     consumes = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE],
     produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
-  fun upsert(@RequestBody deliveryConfig: SubmittedDeliveryConfig): DeliveryConfig =
+  fun upsert(
+    @RequestBody
+    @io.swagger.v3.oas.annotations.parameters.RequestBody(description = "The delivery config. If its `name` matches an existing delivery config the operation is an update, otherwise a new delivery config is created.")
+    deliveryConfig: SubmittedDeliveryConfig
+  ): DeliveryConfig =
```

Comment:
Example of using swagger annotations on a controller. Unfortunate that here it has the same name as the Spring one.

Reply:
I guess we could create our own annotation that combines the two

Reply:
Actually using our own annotations in general might be a good way round the problem of not wanting to pull dependencies into `keel-api`